### DISCUSSION TITLE
German Umlaute are handled as InvalidInputException

### DISCUSSION THREAD

**Discussion by dimkaart** [duckdb/duckdb#10289] [Q&A]
I wanted to read a .csv file with german umlaute (äöü) but duckdb returns the InvalidInputException error until I transformed all umlaute. Is there any other solution?

**Comment by Mause** [comment]
Are you able to provide a sample file?

**Reply by dimkaart** [reply]
test.csv:

row1; row2
1; 2
test; Küche
3; 4

**Reply by EbiArnie** [reply]
This should work if the file is UTF-8 encoded - at least it does work for me, with DuckDB 0.9.2.
In your case, it's probably latin1 encoded.
If this is the case, you could try to convert it first; e.g. with the iconv command line tool:
`iconv -f iso-8859-15 -t utf-8 test.csv > test.utf8.csv`
